components/speedreader/rust/lib/src/readability/src/statistics.rs

```diff
@@ -267,7 +267,14 @@ pub fn collect_statistics(dom: &Sink) -> Option<ReadableFeatures> {
     let head = dom::document_head(&dom)?;
     let body = dom::document_body(&dom)?;
     features.is_open_graph_article = is_open_graph_article(&head);
-    collect_scores(&body, &mut features);
+    let paragraph_len_threshold = if features.is_open_graph_article {
+        PARAGRAPH_LENGTH_THRESHOLD
+    } else {
+        // Double the paragraph length to 4 sentences if not explicitly tagged
+        // as an article.
+        PARAGRAPH_LENGTH_THRESHOLD * 2
+    };
+    collect_scores(&body, paragraph_len_threshold, &mut features);
```

Comment:
I'd imagine it'd be useful to be able to tweak these independently of each other - personally I'd either make these into separate `const`s, or even an enum with an `Into<usize>` impl that you can pass to `collect_scores` rather than passing the number directly (since there's two distinct classifications).